.github/workflows/ci.yaml

```diff
@@ -80,8 +80,8 @@ jobs:
     - name: Create kube-prometheus stack
       run: |
         kubectl create -f manifests/setup
-        until kubectl get servicemonitors --all-namespaces ; do date; sleep 1; echo ""; done
```

Comment:
This was here to check if servicemonitor CRD was applied correctly and to prevent issues you can see in CI - https://github.com/prometheus-operator/kube-prometheus/runs/4773482302?check_suite_focus=true#step:5:108